Bottom: 923a59f751ef5a438528586054bc60ead4a003fc
Top:    01b56d3f125c467cd8798de62734d65f1e912b0f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 17:41:47 +0000

Refresh of hid-gtk-fix-netlistchanged-act

---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index 922a03c..249066f 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -1009,7 +1009,9 @@ ghid_netlist_window_update (gboolean init_nodes)
 static gint
 GhidNetlistChanged (int argc, char **argv, Coord x, Coord y)
 {
-  if (!ghidgui->is_up)
+  /* XXX: We get called before the GUI is up when
+   *         exporting from the command-line. */
+  if (ghidgui == NULL || !ghidgui->is_up)
     return 0;
 
   loading_new_netlist = TRUE;
